appservice/src/main/java/nu/yona/server/subscriptions/rest/BuddyController.java

```diff
@@ -158,11 +166,11 @@ public BuddyResource(BuddyDTO buddy)
 			}
 
 			HashMap<String, Object> result = new HashMap<String, Object>();
-			result.put(BuddyDTO.USER_REL_NAME,
-					new UserController.UserResourceAssembler(false).toResource(getContent().getUser()));
+			result.put(curieProvider.getNamespacedRelFor(BuddyDTO.USER_REL_NAME),
```

Comment:
Indeed, provided Spring HATEOAS it is a link. Here, we have a property with the name ``_embedded``, of type ``map``. There doesn't know it needs to do the replacement.